specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/examples/Projects_AccessArtifacts.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
```

Comment:
This file not required for public preview.

Reply:
Removed.